.github/workflows/routine.yaml

```diff
@@ -22,6 +22,10 @@ on:
         type: boolean
         default: true
         required: false
+      minimize-man-figures:
```

Comment:
What do you think about renaming this `minimize-figures` and allowing the input to be a comma- or newline-separated list of folders, along the lines of the suggestion in https://github.com/rstudio/shiny-workflows/issues/14?

It'd be helpful to also minimize figures in `vignettes` and sometimes images show up in nonstandard places in pkgdown sites. 

That said, I think

```
minimize-figures: "man/figures/**.png, vignettes/**.png"
```

would be a reasonable default.

Reply:
Could it take a list as a value? Also, I slightly prefer the name `optimize-pngs` because that makes it very clear that it only affects PNG files, and not, say, JPEG.


```
optimize-pngs: ["man/figures/**.png", "vignettes/**.png"]
```

Reply:
Name change: 👍 
List format: 👎 

How about a multi line string?

```
optimize-pngs: |
  man/figures/**.png
  vignettes/**.png
```